setup.py

```diff
@@ -53,6 +53,6 @@
     test_suite="tests",
     tests_require=test_requirements,
     url="https://github.com/namespace-ee/django-datadog-logger",
-    version="0.7.1",
+    version="0.7.2",
```

Comment:
pull requests should not increment version in this file. this is handled with `bump2version` prior to release.